features/admin.administrators.v1/wizard/steps/admin-user-basic.tsx

```diff
@@ -191,6 +215,15 @@ export const AddAdminUserBasic: React.FunctionComponent<AddAdminUserBasicProps>
         }
     }, [ checkedAssignedListItems ]);
 
+    useEffect(() => {
+        document
+            ?.getElementById(
+                administratorType === AdminAccountTypes.INTERNAL
+                    ? "inviteInternalUserForm"
+                    : "inviteExternalUserForm"
```

Comment:
Shall we include these in a `const` instead of using just the string?

Reply:
Fixed with 7589832